website/docs/getting-started/project-setup.md

```diff
@@ -22,25 +21,29 @@ You can check your toolchain version using `rustup show` (under "active toolchai
 
 Rust can compile source codes for different "targets" (e.g. different processors). The compilation target for browser-based WebAssembly is called "wasm32-unknown-unknown".  The following command will add this target to your development environment.
 
-`rustup target add wasm32-unknown-unknown`
+```shell
+$ rustup target add wasm32-unknown-unknown
+```
 
 ## Install Trunk
 
 Trunk is the recommended tool for managing deployment and packaging, and will be used throughout the documentation and examples.
 See [Wasm Build Tools](./../more/wasm-build-tools.md) for more information on packaging and alternatives.
 
-\```
+```shell
 # note that this might take a while to install, because it compiles everything from scratch
 # Trunk also provides prebuilt binaries for a number of major package managers
 # See https://trunkrs.dev/#install for further details
-cargo install trunk
-\```
+$ cargo install trunk
+```
 
 ## Install wasm-bindgen-cli
 
-Trunk uses a tool called wasm-bindgen-cli to perform deployment and packaging, which should be installed using the following command.
-
+Trunk uses a tool called wasm-bindgen-cli to perform deployment and packaging, which should be installed using the following command:
 
+```shell
+$ cargo install wasm-bindgen-cli
+```
```

Comment:
Let's remove this. Trunk automatically downloads it now

Reply:
Section removed.  Always happy to remove unneeded instructions.